ui/src/app/app.constants.ts

```diff
@@ -101,4 +101,10 @@ export enum ProgressStatus {
 export const DialectList = [
   { value: 'google_standard_sql', displayName: 'Google Standard SQL Dialect' },
   { value: 'postgresql', displayName: 'PostgreSQL Dialect' },
-]
\ No newline at end of file
+]
+
+export const Dataflow = {
+  Network: 'network',
+  Subnetwork: 'subnetwork',
+  IsDataflowConfigSet: 'IsDataflowConfigSet',
```

Comment:
nit: different case for the variables. We should follow a consistent naming structure.

Reply:
Updated